network-store-server/src/main/java/com/powsybl/network/store/server/QueryCatalog.java

```diff
@@ -39,13 +39,16 @@ public final class QueryCatalog {
     static final String INDEX_COLUMN = "index";
     static final String TAPCHANGER_TYPE_COLUMN = "tapChangerType";
     static final String ALPHA_COLUMN = "alpha";
-    static final String OPERATIONAL_LIMITS_GROUP_ID_COLUMN = "operationalLimitsGroupId";
-    static final String SELECTED_OPERATIONAL_LIMITS_GROUP_ID_COLUMN = "selectedOperationalLimitsGroupId";
+    static final String TEMPORARY_LIMITS_TABLE = "newtemporarylimits";
+    static final String TEMPORARY_LIMITS = "temporarylimits";
+    static final String OLD_TEMPORARY_LIMITS = "temporarylimit";
```

Comment:
need to find a better name than old because this code is going to stay if it goes in the liquibase changeset. Maybe "V211LimitMigration"